core/src/consensus/stake/mod.rs

```diff
@@ -892,6 +916,8 @@ mod tests {
         let delegator = public_to_address(&address_pubkey);
 
         let mut state = helpers::get_temp_state();
+        assert_eq!(Ok(()), state.update_params(0, CommonParams::default_for_test()));
```

Comment:
Why don't we add this `state.update_params(0, CommonParams::default_for_test());` to `helpers::get_temp_state();`? or let calling `update_params()` create metadata automatically?
And `state.update_params()` is not a subject for this tests. I think it can just `.unwrap()` rather than `assert_eq!()`

Reply:
There is a test whether the initial state is a root of the empty trie.

Reply:
How about making a local test helper?

Reply:
I think it doesn't have worth to make a helper function.
And please prefer to use `assert_eq` if the argument derives `Debug` and `PartialEq`.
You should read the backtrace if you use `unwrap`, because the test fails in `Option`.

Reply:
Okay then. I'll comment like `// Arrange`, `// Action`, `// Assert` to separate sections to convey the intention of a test in that situation.